### ISSUE TITLE
[filter] Hide images without alt from AT

### ISSUE THREAD

**Issue by erikkroes** [erikkroes/access-tools#29]
Add `alt=""` or `aria-hidden` to images that have no alt text and aren't marked up as decorative.
